www/templates/diary/list.html

```diff
@@ -89,16 +89,38 @@
                     </div>
                 </div>
              </div>
-             <div class="row" style="padding-left: 5px;padding-right: 5px;">
-                <div class="col-50" style="text-align: center;font-size: 14px;font-weight: 600;" translate>
-                {{'.mode'}}
+             <div ng-if="isAnalyzed(tripgj)" class="row trip-confirm-container">
+                  <button
+                    ng-if="tripgj.customSurveyLabel"
+                    class="button btn-trip-confirm btn-trip-confirm-green"
+                    ng-click="confirmSurvey(tripgj)"
+                    >
+                    <b>{{tripgj.customSurveyLabel}}</b>
+                  </button>
+                  <button
+                    ng-if="!tripgj.customSurveyLabel"
+                    class="button btn-trip-confirm btn-trip-confirm-white"
+                    ng-click="confirmSurvey(tripgj)"
+                    >
+                    Add Trip Details
```

Comment:
This looks like it still has the enketo survey logic - the label is `"Add Trip Details"` instead of `{{ .purpose }}`